### DISCUSSION TITLE
Drawing of PWM3360

### DISCUSSION THREAD

**Discussion by Gao0130** [Schievel1/dactyl_manuform_r_track#4] [Q&A] [answered]
This is the only PWM3360 we can found in China. While it has many difference with yours. Do you have any methods to provide a Drawing of PWM3360? Thats the biggest issue right now. 

![PWM3360](https://user-images.githubusercontent.com/60763571/204133409-549c379d-6c28-46c4-97ac-2e91363c262a.png)

Anyway, I may found a piracy product of Arduion Micro, can we use it?

![Arduion Micro](https://user-images.githubusercontent.com/60763571/204133420-191c1b74-dd24-4328-b28a-850fd9590fd5.jpg)

**Comment by Schievel1** [comment, marked as answer]
About the Arduino first, because that's an easier one.
Yes, I think you can use it. But be aware that when it comes to those micro controllers there are many Chinese knock offs that do not work right. I just had an issue with a batch of STM32F411 which I bought from China, and none of them worked. (Couldn't flash firmware)
On the other hand I've been using the Arduino Pro Micros from AliExpress everywhere and I never had an issue. 
So, there is a fair chance it works.

About the PWM. Where can I buy that? ;D I got mine from the US, and shipping is really expensive. If you tell me where I can buy that, I would like to order one and have a look.
Judging from the pins and what it says on the board it would assume it will work. 

Here is a link to the drawing 

https://github.com/mrjohnk/PMW3360DM-T2QU/tree/master

**Reply by Gao0130** [reply]
The author of this gives two versions of the sensor peripheral circuit schematic, but I can't use kicad to draw the board, and the lidron eda I would use doesn't have this component in it.

**Reply by Schievel1** [reply]
> The author of this gives two versions of the sensor peripheral circuit schematic, but I can't use kicad to draw the board, and the lidron eda I would use doesn't have this component in it.

The wiring to the actual sensor is the same, so I don't thing the version matters much.

**Comment by Schievel1** [comment]
Cheers mate! 

I just made some PMW3360 Sensors with the same dimensions like the ones here https://www.tindie.com/products/jkicklighter/pmw3360-motion-sensor/
If you are interested let me know. My wife is flying to China in the summer and I can organize a cheap delivery if you are willing to wait a few months.

**Reply by Gao0130** [reply]
Hey mate, i just want to ask you about do you have zmk hardware of this project, I'm already constructed the PWM3360, and it works well. If you want some cheap 3360 by my hands about 5€ cost price please let me know maybe i can make some for you. Please help me with zmk hardware, I'm trying to improve to Bluetooth edit!
